docs/labs/README.md

```diff
@@ -0,0 +1,193 @@
+# AKS MCP Transport Testing Guide
```

Comment:
nit: calling Lab instead of test/testing in this document (it is a user-facing lab instead of a test of this project)